zone/bot_commands/cast.cpp

```diff
@@ -543,6 +546,9 @@ void bot_command_cast(Client* c, const Seperator* sep)
 				tar = bot_iter;
 			}
 
+			bool los_required = bot_iter != tar && !IsAnyHealSpell(chosen_spell_id) && !IsPBAESpell(chosen_spell_id);
```

Comment:
Didn't we just add a helper to simplify if a spell requires LOS ? It'd be good to centralize that into a readable function

Reply:
https://github.com/EQEmu/Server/pull/4746/files#diff-417633ac61d6e8de2dd9af8c0f76b6e7d0cf394bcb8b9171484527361c98455fR528

Reply:
Current helper requires a spell_type as well, created one just for spell_id.